### ISSUE TITLE
Add support for rendering LaTeX in text/annotations

### ISSUE THREAD

**Issue by kristofk01** [ray-pH/diagramatics#6]
Hi again,

I think it would be pretty useful to allow LaTeX rendering on text and/or annotation elements. 

For example
```
text("f(x)=a^3+b^2+c")
```
should render something like
![image](https://github.com/ray-pH/diagramatics/assets/23499974/80818ab5-9fc5-4e2b-a26f-51960e5136c2)

To my knowledge the existing API only allows rendering in the "math variable italic style" and does not render to LaTeX.

I'm loving this library btw

Cheers.

**Comment by ray-pH** [comment]
This might be possible using something like MathJax. I'm pretty sure there is a simple way to make it work on the browser, but i'm not sure how it'll interact with other objects and whether the MathJax object can be downloaded as svg or not.

I'll try to figure it out, working on it.

In the meantime, you can use unicode to achieve that `"f(x) = a³ + b² + c"`. I know this is a hassle and supporting LaTeX expression is definitely better.

**Comment by ray-pH** [comment]
Sorry for the long wait, i had a lot going on in the last week

I've implemented the functionality necessary to do that in #8, it should be available in v1.1.0

I still want to make the library separate and not depend on another library like MathJax, so what you can do is import MathJax separately, and provide `MathJax.tex2svg()` into `handle_tex_in_svg()`.

in your HTML, you can import MathJax (tex-svg) separately
```html
<script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js"></script>
```

and in the script, create a function `handletex` that given a tex string and a config, return an SVG element in form of a string. In `draw`, you can then call `handle_tex_in_svg(svg, handletex)` after `draw_to_svg(...)`
```js
let handletex = (str, conf) => {
    return MathJax.tex2svg(str, conf).innerHTML;
}
let draw = (...diagrams) => {
    draw_to_svg(svgelem, diagram_combine(...diagrams));
    handle_tex_in_svg(svgelem, handletex);
}
```

Diagramatics will recognize any text object in the form `"$...$"` (or `"$$...$$"` for display style) as a tex object.
```js
let tx = text('$a^2+b^2$').fontsize(24);
```

I haven't had time to write documentation for it yet. For now, you can take a look at this working example on JSitor: [MathJax Example](https://jsitor.com/DJqa7rQPlW).

**Comment by kristofk01** [comment]
Thanks very much
